### PR TITLE
Make tooltip aria-hidden and remove aria-expanded on calling element

### PR DESCRIPTION
**Pull request by pascalwengerter**

### CODE REVIEW

File: src/directives/OcTooltip.js [hunk 1]
```diff
@@ -49,9 +61,11 @@ export default {
       ignoreAttributes: true,
       ...(Object.prototype.toString.call(value) === "[object Object]" && value),
       aria: {
-        content: "describedby",
+        content: null,
+        expanded: false,
+        hidden: true,
```
Comment (fschade) [inline src/directives/OcTooltip.js:66]: this can go because tippy doesn't support the option and we manage it our own by ariaHidden plugin

**Review by fschade — COMMENTED** [review]
just one change, expect that 🚀

File: src/directives/OcTooltip.js (hunk 1 above)
Comment (pascalwengerter) [reply src/directives/OcTooltip.js]: Thanks, should be resolved!

**Review by fschade — APPROVED** [review]
LGTM 🚀

**Pull request merged**